### ISSUE TITLE
package httpuv missing from dependencies

### ISSUE THREAD

**Issue by allonw** [YosefLab/scone#60]

**Comment by drisso** [comment]
Maybe I'm missing something, where are we using the package httpuv?

@mbcole do you know?

**Comment by allonw** [comment]
This is the error I got while installing:

* installing *source* package 'scone' ...
** R
** data
** inst
** tests
** preparing package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called 'httpuv'
ERROR: lazy loading failed for package 'scone'

**Comment by drisso** [comment]
It's not our dependency. Are you using the `dependencies=TRUE` option?

**Comment by drisso** [comment]
It may be a dependency of our dependency. Have you tried to manually install it?

**Comment by allonw** [comment]
Yes, I was using  `dependencies=TRUE` because that's what the front page recommends :)

https://github.com/YosefLab/scone

**Comment by drisso** [comment]
Yes! And I stand behind that! You should use it :)

**Comment by allonw** [comment]
PS. I'm sorry if it wasn't clear from the title of the issue: yes, I installed httpuv manually and then everything worked fine.

**Comment by mbcole** [comment]
Would `recursive = TRUE` have worked automatically?

**Comment by drisso** [comment]
can I close this one? Or should we change the README to include the recursive=TRUE?

**Comment by allonw** [comment]
Whatever you think is right.

My case was that I used recursive=TRUE as appears in the readme and yet of the required packages needed manual installation. Once I installed it, scone's installation went smoothly

**Comment by drisso** [comment]
In the README there is dependencies=TRUE, but not recursive=TRUE. Did you use recursive?

**Comment by allonw** [comment]
Sorry, my error. I meant to write that I used dependencies=TRUE as in the readme. I didn't use recusive=TRUE

**Comment by allonw** [comment]
Updated the issue's name to reflect the problem better (also found it applies to package "digest")

**Comment by drisso** [comment]
@allonw @WadeDavis are you both on Windows?

**Comment by allonw** [comment]
When I reported this issue I encountered in Windows. I think I've since saw it in Linux, but it's been a while ago so I'm not sure if I recall correctly by now.

**Comment by drisso** [comment]
I believe this is an issue with devtools. There are many open issues there that could be related, e.g.,
https://github.com/hadley/devtools/issues/1298

I would try the following:

1. Run devtools directly (biocLite uses devtools internally) `devtools::install_github("YosefLab/scone", force = TRUE)` and/or
2. Update to the most recent version of devtools and try again

**Comment by AnneGeijsen** [comment]
Hi!

Were you able to solve the problem? I have the same problem and I tried all suggestions in this post and several others but I am not able to solve the problem.
```
library("devtools")
devtools::install_github("rwehrens/BatchCorrMetabolomics", force = TRUE)
```
and 
```
library("devtools")
install_github("rwehrens/BatchCorrMetabolomics")
```
 I also tried to change `.libPaths() ` but nothing seems to work.. I am working in Windows 10.

Thanks a lot!

**Comment by drisso** [comment]
Hi @AnneGeijsen ,

this is the issue tracker of the `scone` package, I believe that you have a problem with `devtools` not `scone`. If so, you should post it here: 
https://github.com/hadley/devtools/issues
